CHANGELOG.md

```diff
@@ -124,6 +124,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 - We fixed an issue when the preview was out of sync. [#9172](https://github.com/JabRef/jabref/issues/9172)
 - We fixed an issue where identifier paste couldn't work with Unicode REPLACEMENT CHARACTER. [#11986](https://github.com/JabRef/jabref/issues/11986)
 - We fixed an issue when click on entry at "Check Integrity" wasn't properly focusing the entry and field. [#11997](https://github.com/JabRef/jabref/issues/11997)
+- We Fixed an error where saving a non-absolute URI threw an IllegalArgumentException. Now, it checks if the URI is absolute and throws a clear error message if it is not.[#12186](https://github.com/JabRef/jabref/issues/12186)
```

Comment:
```suggestion
- On saving a non-absolute URI, now a clear error message is shown. [#12186](https://github.com/JabRef/jabref/issues/12186)
```

Reply:
Changelog is primarily for the users. This would be too long and technical for the user to understand.